### PR TITLE
Branch activity delete command

### LINKED ISSUE #43: add ActivityDeleteCommand class
To delete an Activity object as specified by the user input.

### LINKED ISSUE #95: As a user, I can delete activities in sessions that were previously created
... so that I can undo a mistake in activity creation.

### PR DESCRIPTION
**Pull request by ivanaitzliddat**
Add relevant and necessary methods for ActivityDeleteCommand class
Add Javadoc for all methods
Update getCommand method in Parser class

Fix #43 
Fix #95

### CODE REVIEW

File: src/main/java/seedu/splitlah/command/ActivityDeleteCommand.java [hunk 1]
```diff
@@ -24,8 +38,52 @@ public ActivityDeleteCommand(int sessionId, int activityId) {
         this.activityId = activityId;
     }
 
+    /**
+     * Prepares user arguments for activity delete command.
```
Comment (froststein) [inline src/main/java/seedu/splitlah/command/ActivityDeleteCommand.java:42]: Consider following the standard the group has set `Prepares user arguments for the deletion of a <Command Name> object.`

**Review by froststein — CHANGES_REQUESTED** [review]
Change javadocs

File: src/main/java/seedu/splitlah/command/ActivityDeleteCommand.java [hunk 2]
```diff
@@ -24,8 +38,52 @@ public ActivityDeleteCommand(int sessionId, int activityId) {
         this.activityId = activityId;
     }
 
+    /**
+     * Prepares user arguments for the deletion of an Activity object.
```
Comment (froststein) [inline src/main/java/seedu/splitlah/command/ActivityDeleteCommand.java:42]: Creation for Command object not activity object

**Review by froststein — CHANGES_REQUESTED** [review]
Please check again on the javadocs standard the team is set.

**Review by froststein — APPROVED** [review]
Looks good to merge

**Review by lelerer — APPROVED** [review]
Looks good to merge.

**Pull request merged**